qemu/tests/cfg/qemu_guest_agent_hotplug.cfg

```diff
@@ -10,7 +10,11 @@
     gagent_status_cmd = "service qemu-guest-agent status"
     gagent_pkg_check_cmd = "rpm -q qemu-guest-agent"
     gagent_check_type = sync
-    extra_params += "-device virtio-serial-pci,id=virtio-serial0,bus=pci.0"
+    i440fx:
```

Comment:
Better remove this line and keep L#13 as it was, since you support ppc64 as well in L#3.